Makefile

```diff
@@ -96,7 +96,7 @@ docker-push-release:  docker-tag-release
 docker-build:
 	docker build . -t ${IMG}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/manager_image_patch.yaml
```

Comment:
Is it possible for manager_image_patch.yaml to be an empty file, then when we run docker build, populate it via `echo >`?

Reply:
@maxsmythe I've moved the manager_image_patch.yaml lines to the Makefile to be echoed upon running docker build, per our discussion.